packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx

```diff
@@ -73,47 +66,44 @@ const Header: FC<IHeaderProps> = () => {
   };
 
   return (
-    <NavHeader.Root brand={isMediumScreen ? 'DevTools' : 'Kadena'}>
-      <NavHeader.Navigation activeHref={pathname}>
+    <NavHeader
+      logo={
+        <Link href={'/'}>
+          <Logo />
```

Comment:
We used to have two logos for smaller/larger screens, so this isn't the case anymore?

Reply:
That was unintentional! I will fix it 👍